src/main/java/com/social/instagram/exception/DuplicateUserIdException.java

```diff
@@ -1,9 +1,5 @@
 package com.social.instagram.exception;
 
-public class DuplicateUserIdException extends IllegalArgumentException {
-
-    public DuplicateUserIdException(String message) {
-        super(message);
-    }
+public class DuplicateUserIdException extends IllegalArgumentException{
```

Comment:
여기도 띄어쓰기를 하나 해주는게 좋겠네요. `intellij checkstyle`을 검색해보시고, 적절한 컨벤션을 만들어서 써보셔도 좋을 것 같아요

```suggestion
public class DuplicateUserIdException extends IllegalArgumentException {
```